src/plugins/geomap/_base.scss

```diff
@@ -324,3 +315,19 @@ button {
 		z-index: 10000;
 	}
 }
+
+.geomap-aoi {
+	legend {
+		border: 0;
+		font-size: 1em;
+		margin-bottom: 1em;
+	}
+
+	button {
+		&.geomap-geoloc-aoi-btn {
+			position: absolute;
+			top: auto;
+			right: 15px;
+		}
+	}
+}
```

Comment:
SCSS requires a new line at the of the file (You can maybe configure your editor to do this automatically)